api-services/offline/tasks/bed_service.ts

```diff
@@ -144,25 +139,6 @@ export class BedOfflineServicePromiseClient extends BedServicePromiseClient {
     return new CreateBedResponse().setId(newBed.id)
   }
 
-  async bulkCreateBeds(request: BulkCreateBedsRequest, _?: Metadata): Promise<BulkCreateBedsResponse> {
-    const roomId = request.getRoomId() // TODO check existence
-    const beds: BedWithRoomId[] = range(0, request.getAmountOfBeds() - 1).map(value => ({
-      id: Math.random().toString(),
-      name: `Bed ${value + 1}`,
-      roomId,
-    }))
-
-    for (const bed of beds) {
-      BedOfflineService.addBed(bed)
-    }
-
-    const list = beds.map(bed => new BulkCreateBedsResponse.Bed()
-      .setId(bed.id)
-      .setName(bed.name))
-
-    return new BulkCreateBedsResponse().setBedsList(list)
-  }
-
```

Comment:
Why did you remove this? The method still exists on the API

Reply:
Removed since https://github.com/helpwave/services/commit/dcc524f1165b117d833803f8a874373440016cee#diff-2fa17908774eb944a99da99eeea50dcb5eb50a134388d74898306e2bac1534f2L20

Reply:
cc. @FoseFx